samples/system-test/buckets.test.js

```diff
@@ -59,6 +59,9 @@ async function deleteAllBucketsAsync() {
 }
 
 after(deleteAllBucketsAsync);
+afterEach(async () => {
+  await new Promise(res => setTimeout(res, 1000));
```

Comment:
Is 1s enough?

Reply:
Should be, looks to be failing due to rapid bucket metadata updates. Docs say one update per second.